docs/getting-started.md

```diff
@@ -58,17 +58,12 @@ decompress it (such as [7-Zip] on Windows).
     If you do not want to lose them!
 </div>
 
-There are a vast number of ways to do this, so pick the one from the list that
-sounds the easiest to you:
+Now it's time to write the image to the card. The easiest way is to follow our
+guide to using the cross-platform tool Etcher [here](/docs/tutorials/writing-sd-card-image-etcher).
 
-* **Linux**
-    * [Command Line](../tutorials/writing-sd-card-image-linux-command-line)
-    * [Ubuntu (graphical interface)](../tutorials/writing-sd-card-image-ubuntu-disk-image-writer)
-* **OS X**
-    * [Command Line](../tutorials/writing-sd-card-image-osx-command-line)
-    * [Apple Pi Baker (graphical interface)](../tutorials/writing-sd-card-image-osx-apple-pi-baker)
-* **Windows**
-    * [Win32DiskImager (graphical interface)](../tutorials/writing-sd-card-image-win32diskimager)
+If you would prefer to do it manually or the recommended option does not work
```

Comment:
Do something that isn't simply flashing the image with Etcher, whatever that may be.